staging/src/k8s.io/apimachinery/pkg/api/resource/quantity.go

```diff
@@ -705,7 +705,7 @@ func (q *Quantity) MilliValue() int64 {
 	return q.ScaledValue(Milli)
 }
 
-// ScaledValue returns the value of ceil(q * 10^scale); this could overflow an int64.
+// ScaledValue returns the value of ceil(q / 10^scale); this could overflow an int64.
```

Comment:
As per https://github.com/kubernetes/kubernetes/blob/master/vendor/gopkg.in/inf.v0/dec.go#L41 the scale is the negated form of the exponent. That seems to line up with AsScaledInt64() which will call negativeScaleInt64 for a value with a scale of 0 and an explicit scale greater than 0. negativeScaleInt64 will then divide by 10 for each integer of scale.